packages/editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -11,7 +18,16 @@ const FormatToolbar = ( { controls } ) => {
 				{ controls.map( ( format ) =>
 					<Slot name={ `RichText.ToolbarControls.${ format }` } key={ format } />
 				) }
-				<Slot name="RichText.ToolbarControls" />
+				<Slot name="RichText.ToolbarControls">
+					{ ( fills ) =>
```

Comment:
Should `DropdownMenu` be conditionally included only when there are fills to render?